certified-connectors/Delinea Secret Server/apiDefinition.swagger.json

```diff
@@ -0,0 +1,741 @@
+            "description": "Secret Server authentication password.  Required when authenticating"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful retrieval of an access token",
+              "schema": {
+               "$ref": "#/definitions/TokenResponse"
+            }
+          },
+          "400": {
+            "description": "An error occurred",
+            "schema": {
+              "$ref": "#/definitions/TokenErrorResponse"
+            }
+          }
+        } 
+      }        
+    },
+    "/api/v1/secrets/{id}": {
+      "get": {
+        "description": "Get a single secret by ID",
+        "operationId": "GetSecretById",
+        "summary": "Get secret",
+        "parameters": [
+          {
+            "x-ms-summary": "Secret ID",
+            "description": "Secret ID for get secret object",
+            "x-ms-url-encoding": "single",
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "type": "integer"
+          },
+          {
+            "x-ms-summary": "Authorization",
+            "description": "Bearer <Access-Token>",
+            "in": "header",
+            "name": "Access-Token",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Secret object",
+            "schema":{
+              "$ref": "#/definitions/SecretModel"
+            }
+          },
+          "400":{
+            "description": "Bad request",
+            "schema":{
+              "$ref": "#/definitions/BadRequestResponse"
+            }
+          },
+          "403": {
+            "description": "Authentication failed",
+            "schema": {
+              "$ref": "#/definitions/AuthenticationFailedResponse"
+            }
+          },
+          "500": {
+            "description": "Internal server error",
+            "schema": {
+              "$ref": "#/definitions/InternalServerErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/api/v1/secret-templates/{id}": {
+      "get": {
+        "description": "Get a single secret template by ID",
+        "operationId": "GetTemplateById",
+        "summary": "Get secret template",
+        "parameters": [
+          {
+            "x-ms-summary": "Template ID",
+            "description": "Get template secret by Id",
+            "x-ms-url-encoding": "single",
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "type": "integer"
+          },
+          {
+            "x-ms-summary": "Authorization",
+            "description": "Bearer <Access-Token>",
+            "in": "header",
+            "name": "Access-Token",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Secret template object",
```

Comment:
@tylerezimmerman please look through 2xx response codes and provide another description for them. Just OK would be enough here, above you have `Successful retrieval of an access token` which also looks good.